shoop/themes/classic_gray/views/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .product_preview import product_quick_view as preview
+from .basket import basket_partial
```

Comment:
Ah, and to appease the vengeful spirit that is flake8, you'll also have to add

```python

__all__ = ["preview", "basket_partial"]

```

here.